_data/chains/eip155-6398.json

```diff
@@ -9,7 +9,7 @@
   },
   "faucets": [],
   "infoURL": "",
-  "shortName": "connext-sepolia",
+  "shortName": "ESN",
   "chainId": 6398,
   "networkId": 6398,
   "explorers": [
```

Comment:
optional suggestion;
Adjust the name and icon since the referring one isn't match the existing one on landing page and icon still the connext one not everclear